packages/eslint-config-typescript/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@side/eslint-config-typescript",
+  "version": "0.0.0",
+  "main": "eslint-config-typescript.js",
+  "files": [
+    "eslint-config-typescript.js"
+  ],
+  "license": "UNLICENSED",
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/reside-eng/lint-config.git"
+  },
+  "bugs": {
+    "url": "http://github.com/reside-eng/lint-config/issues"
+  },
+  "dependencies": {
+    "@typescript-eslint/eslint-plugin": "5.18.0",
+    "@typescript-eslint/parser": "5.18.0"
+  },
+  "peerDependencies": {
+    "eslint": ">=7.14.0",
+    "typescript": ">=4.0.0"
+  },
+  "prettier": "@side/prettier-config"
```

Comment:
How does this work? Where are we pulling in this dependency? It can start to get a little confusing if we're using a dependency of ourself in the same package like this. Maybe we can point to the local package instead?

Reply:
You are right – I thought about it a bit more and the prettier config isn't needed for this.